modulefiles/module_gwci.orion.lua

```diff
@@ -2,21 +2,14 @@ help([[
 Load environment to run GFS workflow ci scripts on Orion
 ]])
 
-prepend_path("MODULEPATH", "/apps/contrib/NCEP/hpc-stack/libs/hpc-stack/modulefiles/stack")
+prepend_path("MODULEPATH", "/work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.5.1/envs/gsi-addon/install/modulefiles/Core")
 
-load(pathJoin("hpc", "1.1.0"))
-load(pathJoin("hpc-intel", "2018.4"))
-load(pathJoin("hpc-impi", "2018.4"))
-load(pathJoin("netcdf","4.7.4"))
-load(pathJoin("netcdf","4.7.4-parallel"))
-load(pathJoin("nccmp"," 1.8.7.0"))
-load(pathJoin("contrib","0.1"))
-load(pathJoin("wgrib2","3.0.2"))
+load(pathJoin("stack-intel", os.getenv("2022.0.2")))
+load(pathJoin("stack-intel-oneapi-mpi", os.getenv("2021.5.1")))
 
-prepend_path("MODULEPATH", "/work2/noaa/global/wkolczyn/save/hpc-stack/modulefiles/stack")
-load(pathJoin("hpc", "1.2.0"))
-load(pathJoin("hpc-intel", "2018.4"))
-load(pathJoin("hpc-miniconda3", "4.6.14"))
-load(pathJoin("gfs_workflow", "1.0.0"))
+load(pathJoin("netcdf-c", os.getenv("4.9.2")))
+load(pathJoin("netcdf-fortran", os.getenv("4.6.0")))
+load(pathJoin("nccmp","1.9.0.1"))
+load(pathJoin("wgrib2", "2.0.8"))
```

Comment:
Do we not need the `py-yaml` etc here from `gfs-workflow`?

Reply:
Yea I had this in the wings for validation scripts I have for later but nothing else.  What do we mean about **py-yaml** though?

Reply:
@TerrenceMcGuinness-NOAA spack-stack doesn't load a Python environment like hpc-stack's `gfs-workflow`.  Instead, individual Python modules are loaded when needed.  For the `setup_*` scripts, these include `py-pyyaml`, `py-jinja2`, and `py-numpy`.  If you know what you need, I can add them in, otherwise I will leave them as is for now.